packages/configuration-web-recommended/README.md

```diff
@@ -8,13 +8,11 @@ This package will automatically install all the missing dependencies.
 To use it you will have to install it via `npm`:
 
 ```bash
-npm install @hint/configuration-web-recommended
+npm install @hint/configuration-web-recommended --save-dev
```

Comment:
Shouldn't this use the same text we're using for hints that are included by default? Same for `configuration-development`, etc.
> This package is installed automatically when adding webhint to your project
so running the following is enough:
> ```bash
> npm install hint --save-dev
> ```